info_terminal/infremen/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(infremen)
+
+find_package(catkin REQUIRED COMPONENTS  mongodb_store roscpp actionlib_msgs actionlib) 
```

Comment:
*Missing a lot of dependencies from the package.xml
* `actionlib_msgs` and `actionlib` are not necessary.